ovn_cluster.sh

```diff
@@ -929,8 +1063,10 @@ case "${1:-""}" in
             GW_NAMES+=( "${GW_PREFIX}${i}" )
         done
 
-        for (( i=1; i<=RELAY_COUNT; i++ )); do
-            RELAY_NAMES+=( "${RELAY_PREFIX}${i}" )
+        for (( i=1; i<=CENTRAL_COUNT; i++ )); do
+            for (( j=1; j<=RELAY_COUNT; j++ )); do
+                RELAY_NAMES+=( "${RELAY_PREFIX}${i}-${j}" )
+            done
```

Comment:
To avoid another env var and because relays are per-az I'd change the naming scheme to:
```
for central_name in "${CENTRAL_NAMES[@]}"; do
            for (( j=1; j<=RELAY_COUNT; j++ )); do
                RELAY_NAMES+=( "${central_name}-relay-${j}" )
            done
        done
```

Like this we can get rid of RELAY_PREFIX all together.